backend/services/esweService.js

```diff
@@ -0,0 +1,45 @@
+const { app } = require('../config')
+
+/**
+ * Education skills and work experience (ESWE)
+ */
+class EsweService {
+  #curiousApi = null
```

Comment:
Interesting - is this object style? The problem is.. we don't use these constructs anywhere else in our code base .. it would be interesting to know what this might give us